google/mozc
# Update poduct_icon.icns with larger icon images

## Description

Made poduct_icon.icns include larger icon images to display ConfigDialog and DictionaryTools icons smoothly.

## Issue IDs

https://github.com/google/mozc/issues/964

## Steps to test new behaviors (if any)

 - OS: macOS 
 - Steps:
   1. Build and install Mozc for macOS.
   2. Open launchpad and confirm ConfigDialog and DictionaryTool s' icons.
   If there is no problem, the icons are displayed smoothly as screenshot below.

![pr](https://github.com/google/mozc/assets/1285728/9f926dc1-7056-415e-971f-69b81477be75)


## Additional context

Tested on macOS Sonoma 14.5, Apple M3
